python-package/lightgbm/dask.py

```diff
@@ -438,7 +484,14 @@ def _copy_extra_params(source, dest):
 class DaskLGBMClassifier(LGBMClassifier, _DaskLGBMModel):
     """Distributed version of lightgbm.LGBMClassifier."""
 
-    def fit(self, X, y, sample_weight=None, client=None, **kwargs):
+    def fit(
+        self,
+        X: _DaskCollection,
```

Comment:
Same about inability to pass `dask Series` as above.

Reply:
fixed in https://github.com/microsoft/LightGBM/pull/3866/commits/8b19e04b05f83a38f5213c1103040f49d0a20988